src/actions/breakpoints.js

```diff
@@ -222,7 +229,7 @@ export function setBreakpointCondition(
   return ({ dispatch, getState, client, sourceMaps }: ThunkArgs) => {
     const bp = getBreakpoint(getState(), location);
     if (!bp) {
-      return dispatch(addBreakpoint(location, { condition }));
+      return dispatch(addBreakpoint(location, { condition, hidden: false }));
```

Comment:
`hidden: false` should be the default: 

`dispatch(addBreakpoint(location, { condition }))`